account-kit/signer/src/client/base.ts

```diff
@@ -69,7 +74,8 @@ export abstract class BaseSignerClient<TExportWalletParams = unknown> {
     const { stamper, connection, rootOrgId } = params;
     this.rootOrg = rootOrgId ?? "24c1acf5-810f-41e0-a503-d5d13fa8e830";
     this.eventEmitter = new EventEmitter<AlchemySignerClientEvents>();
-    this.connectionConfig = ConnectionConfigSchema.parse(connection);
+    const connectionConfig = ConnectionConfigSchema.parse(connection);
+    this.alchemyClient = new AlchemySignerClient(connectionConfig);
```

Comment:
Wait so now we have two different signer clients? One base one that platforms implement, but it's functionality lives in a different client?

Seems a bit odd to have the abstract class called client not actually implement the client logic

Reply:
this will be removed in upcoming commit